packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -2368,6 +2374,7 @@ export function getExecuteFunctions(
 				normalizeItems,
 				constructExecutionMetaData,
 			},
+			internalHelpers: getInternalHelperFunctions(additionalData),
```

Comment:
I'm thinking that the name `internalHelpers` is not great. Maybe be more specific? Perhaps `fileHandlingFunctions` and we could even add more in the future?

Since those should be used exclusively by nodes, that seems to make sense IMO.